### DISCUSSION TITLE
Missing GIT_ED editor

### DISCUSSION THREAD

**Discussion by wei** [community/community#26018] [Codespaces]
<p>It looks like neither <code>code</code> or <code>code-insiders</code> are opening.</p>
<pre><code class="lang-auto">$ git commit
hint: Waiting for your editor to close the file... /home/codespace/.local/bin/git-ed.sh: line 9: --wait: command not found
error: There was a problem with the editor '/home/codespace/.local/bin/git-ed.sh'.
Please supply the message using either -m or -F option.
</code></pre>
<p>I’m using my <a href="https://github.com/wei/dotfiles" rel="nofollow noopener">dotfiles</a> to customize the codespaces could I have messed something up? Thank you!</p>

**Comment by Chuxel** [comment, marked as answer]
<p>This appears to be a bug.  Thank you for raising it!!</p>
<p>Run the following from the terminal to fix it:</p>
<pre><code class="lang-auto"> sed -i -e "s/--wait/\$GIT_ED --wait/g" $HOME/.local/bin/git-ed.sh
</code></pre>
